### PR TITLE
docs(rulesets): reorganize chapters

### PR DESCRIPTION
**Pull request by nulltoken**
**Checklist**

- [ ] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nulltoken** [comment]
@P0lip Here's the doco change from #939 in a dedicated PR, as you requested.

File: docs/getting-started/rulesets.md [hunk 1]
```diff
@@ -64,6 +64,47 @@ rules:
 In most cases, you will want to operate on resolved document and therefore won't specify that property.
 You might find `resolved` useful if your rule requires access to $refs.
 
+### Given
+
+The `given` keyword is, besides `then`, the only required property on each rule definition.
```
Comment (philsturgeon) [inline docs/getting-started/rulesets.md]:
```suggestion
The `given` property is one of only two on required properties on each rule definition (the other being `then`). 
```

**Comment by philsturgeon** [comment]
If you accept that change it looks good to me.

File: docs/getting-started/rulesets.md (hunk 1 above)
Comment (nulltoken) [reply docs/getting-started/rulesets.md]: @philsturgeon Fixed

**Pull request merged**